src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -514,14 +534,30 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MP4;
 		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
 			format = FormatConfiguration.MJPEG;
-		} else if (value.contains("div") || value.contains("dx")) {
-			format = FormatConfiguration.DIVX;
+		} else if (
+				value.startsWith("div") ||
+				value.equals("dx50") ||
+				value.equals("dvx1")
+			) {
+				format = FormatConfiguration.DIVX;
+		} else if (value.startsWith("indeo")) { // Intel Indeo Video: IV31, IV32, IV41 and IV50
+			format = FormatConfiguration.INDEO;
+		} else if (value.equals("yuv")) {
+			format = FormatConfiguration.YUV;
+		} else if (value.equals("rgb") || value.equals("rgba")) {
```

Comment:
If see that MediaInfo use them as codecs (not formats) - which I can understand as they probably doesn't have a specific "codec name". They are however only relevant within the format that specifies that they can be used. It's the constraint that these MediaInfo values should only mean this in the specified context I'm requesting, to avoid parsing bugs and general confusion. A "codec" called ```YUV``` doesn't make any sense until you define that it's within the context of the AVI standard, which permits using a stream of raw, uncompressed images with certain color models.